### ISSUE TITLE
Issue with decode the mode first

### ISSUE THREAD

**Issue by senna1128** [JuliaSmoothOptimizers/CUTEst.jl#239]
Hi,

I'm using Julia 1.5.2. I try to run multiple models in parallel using your julia code provided here [https://github.com/JuliaSmoothOptimizers/CUTEst.jl](https://github.com/JuliaSmoothOptimizers/CUTEst.jl). I also pass decode=false.

But I got an error like
```
ERROR: On worker 7:
CUTEst: lib not found; decode problem first
```
This seems CUTEstModel must decode problem first. If I use standard for loop without running in parallel, it works well. So my cutest package should be fine.

Is there any way to fix it? Also I cannot run 
```
fetch_sif_problems()
```
The error is
```
ERROR: UndefVarError: fetch_sif_problems not defined
```
Are they related? Not using fetch_sif_problems does not affect my usage of cutest at the moment, but I'm not sure if it matters in the future.

Best,
Sen

**Comment by abelsiqueira** [comment]
Hi @senna1128, thanks for the report.

You indeed need to decode the problem first using `sifdecoder`. You could decode it at a different time and then call `CUTEstModel`, but at some point the `sifdecoder` has to be called. This is related: https://github.com/JuliaSmoothOptimizers/CUTEst.jl/pull/145#issuecomment-354771226

`fetch_sif_problems` was removed, it's not needed anymore, but we forgot to update the README.

**Comment by abelsiqueira** [comment]
By the way, I forgot we had this information on the README: https://github.com/JuliaSmoothOptimizers/CUTEst.jl#run-multiple-models-in-parallel

**Comment by dpo** [comment]
That's very clever. I also forgot that we could do that!

**Comment by senna1128** [comment]
Hi @abelsiqueira, 

Thank you very much for the quick reply. I tried to run the code
```
addprocs(2)
@everywhere using CUTEst
@everywhere function opt(name, decode)
    nlp = CUTEstModel(name;decode=decode)
    retval = obj(nlp,nlp.meta.x0)
    #retval = nlp.meta.nvar
    finalize(nlp)
    retval
end

probs = sort(CUTEst.select(contype="unc",max_var=5))
map(x->opt(x,true), probs) # Ensure all the problems have been decoded
pmap(x->opt(x,false), probs)
```
provided in #145 comment. I could get the result for  the command```map(x->opt(x,true), probs) # Ensure all the problems have been decoded``` . It returned 95 objective values. While it still reported the same error in  ```pmap(x->opt(x,false), probs)```. Do you think that is my installation issue? I'm using Julia 1.5.2 via JuliaPro.

**Comment by abelsiqueira** [comment]
I've had the same error. I will investigate this today.

**Comment by abelsiqueira** [comment]
Took me a while to get to it, sorry about that. I think the issue is fixed by #242.

**Comment by dpo** [comment]
@senna1128 could you confirm if your isse has been fixed?

**Comment by dpo** [comment]
Please feel free to open a new issue if problems persist.
